src/webgpu/api/operation/resource_init/check_texture/by_ds_test.ts

```diff
@@ -82,19 +82,19 @@ function getStencilTestEqualPipeline(
         }
         `,
       }),
+      targets: [
```

Comment:
nit: the definition of targets could be on a single line.

Reply:
Done, thanks!